mailpoet/tests/integration/Newsletter/ViewInBrowser/ViewInBrowserControllerTest.php

```diff
@@ -237,10 +237,7 @@ public function testItResetsQueueForAutomaticEmailsInPreview() {
   }
 
   public function _after() {
-    $this->truncateEntity(NewsletterEntity::class);
-    $this->truncateEntity(SubscriberEntity::class);
-    $this->truncateEntity(ScheduledTaskEntity::class);
-    $this->truncateEntity(SendingQueueEntity::class);
+    parent::_after();
     // reset WP user role
     $wpUser = wp_get_current_user();
     $wpUser->add_role('administrator');
```

Comment:
And when at it, maybe even resetting the WP users table to some defaults can be useful. I'm sharing these thoughts as I see them, but I think we can extract all of that into a follow-up ticket.